docs/azure_arc_jumpstart/azure_arc_video_indexer/aks/azure_arc_video_indexer_az/_index.md

```diff
@@ -0,0 +1,225 @@
+                            --config "frontend.endpointUri=${dnsName}" 
+
+```
+
+There are some additional Parameters that can be used in order to have a fine grain control on the extension creation
+
+| Parameter | Default | Description |
+|-----------|---------|-------------|
+| AI.nodeSelector | - | The node Selector label on which the AI Pods (speech and translate)  will be assigned to |
+| speech.resource.requests.cpu | 1 | The requested number of cores for the speech pod |
+| speech.resource.requests.mem | 2Gi | The requested memory capactiy for the speech pod |
+| speech.resource.limits.cpu | 2 | The limits number of cores for the speech pod. must be > speech.resource.requests.cpu  |
+| speech.resource.limits.mem | 3Gi | The limits memory capactiy for the speech pod. must be > speech.resource.requests.mem  |
+| translate.resource.requests.cpu | 1 | The requested number of cores for the translate pod |
+| translate.resource.requests.mem | 16Gi | The requested memory capactiy for the translate pod |
+| translateeech.resource.limits.cpu | -- | The limits number of cores for the translate pod. must be > translate.resource.requests.cpu  |
+| translate.resource.limits.mem | -- | The limits memory capactiy for the translate pod. must be > translate.resource.requests.mem  |
+| videoIndexer.webapi.resources.requests.cpu | 0.5 | The request number of cores for the web api pod  |
+| videoIndexer.webapi.resources.requests.mem | 4Gi | The request memory capactiy for the web api pod  |
+| videoIndexer.webapi.resources.limits.cpu | 1 | The limits number of cores for the web api pod  |
+| videoIndexer.webapi.resources.limits.mem | 6Gi | The limits memory capactiy for the web api pod  |
+| videoIndexer.webapi.resources.limits.mem | 6Gi | The limits memory capactiy for the web api pod  |
+| storage.storageClass | "" | The storage class to be used |
+| storage.useExternalPvc | false | determines whether an external PVC is used. if true, the VideoIndexer PVC will not be installed |
+
+example deploy script :
+
+```shell
+                        az k8s-extension create --name videoindexer \
+                            --extension-type Microsoft.videoindexer \
+                            .......
+                            
+                            --config AI.nodeSelector."beta\\.kubernetes\\.io/os"=linux
+                            --config "speech.resource.requests.cpu=500m" \
+                            --config "speech.resource.requests.mem=2Gi" \
+                            --config "speech.resource.limits.cpu=1" \
+                            --config "speech.resource.limits.mem=4Gi" \
+                            --config "videoIndexer.webapi.resources.requests.mem=4Gi"\
+                            --config "videoIndexer.webapi.resources.limits.mem=8Gi"\
+                            --config "videoIndexer.webapi.resources.limits.cpu=1"\
+                            --config "storage.storageClass=azurefile-csi" 
+
+```
+
```

Comment:
show screenshot with success. Also be sure to show an example that uses every parameter in an example basic configuration. Dont assume user will know which values to use for any parameter and explicitly show what will work with the recommended setup presented in the guide.